68-PA-T-49A.md

```diff
@@ -0,0 +1,157 @@
+   ground rule that on this mssion the AGS should be maintained in that 
+   state which makes it most useful to perform the rendezvous in the 
+   event of PNGCS failure. It was noted that if after having established 
+   the preferred techniques in accordance with that ground rule it is 
+   possible to include some AGS systems tests without jeopardizing crew 
+   safety or other mission objects, they would be considered.
+
+2. Nominal situation: PNGCS seams to be working properly and is prime; 
+   AGS must be maintained in optimum state to take over in the event the 
+   PNGSCS fails. This applies to all maneuvers---CSI, CDH, TPI.
+
+   (a). Checking of the PNGCS will be by comparison with the ground 
+   computed solution *only*. That is comparisons of maneuver targeting 
+   from other sources, such as the AGS, backup charts or the CSM will 
+   not be made to commit to the PNGCS. The PNGCS solution will be used 
+   providing it is within acceptable limits of the MSFN solution. One 
+   possible exception here is that, since the CSM optics provide very 
+   strong solutions to the TPI delta V components perpendicular to the 
+   line of sight, comparison with them may be advantageous.
+
+   (b). The state vectors in the AGS will be updated each time PNGCS is 
+   confirmed to be acceptable. This will likely be at each time it is 
+   committed to make the next maneuver using the PNGCS.
+
+   (c). AGS alignments will be made each time the PNGCS is realigned and 
+   each time the state vector in the AGS is updated from the PNGCS.
+
+   (d). No radar data will be input into the AGS as long as the PNGCS is 
+   working. In effect it is obtaining benefit of the radar via the PNGCS 
+   state vector updates since the PNGCS is processing the radar data.
+
+   (e). There is no need to prepare or learn to use backup charts for 
+   CSI and/or CDH maneuvers for this mission. Terminal phase charts are 
+   essential in the LM.
+
+3. In the event of a PNGCS failure: CSI and CDH only.
+
+   (a). For the CSI and CDH, use the AGS in almost the identical manner 
+   in which the Gemini spacecraft was flown .That is, use ground 
+   targeted maneuvers executed with the AGS External delta V mode.
+
+   (b). No radar data would be input into the AGS prior to the CSI and 
+   CDH.
+
+4. By far the most extensive discussion dealt with the Terminal Phase 
+   Initiation (TPI) maneuver and subsequent midcourse maneuvers in the 
+   event of a PNGCS failure, but with the radar still working. This was 
+   the one area still lacking agreement. It is all based on the 
+   assumption that a rendezvous radar failure is obvious as opposed to 
+   insidious. Clarke Hackler (GCD) and Al Nathan (GAEC) where given the 
+   action item to determine if this assumption is reasonable. Our 
+   alternative plans for TPI---PNGCS out, radar working---are as 
+   follows:
+
+   (a). Compare the onboard chart solution for TPI with the MSFN. If the 
```

Comment:
The original does no have periods after these right-parens. Can you tidy that up?